src/mminou.c

```diff
@@ -65,29 +72,83 @@ long g_screenWidth = MAX_LEN; /* Width default = 79 */
 long g_screenHeight = SCREEN_HEIGHT; /* Default = 23 */
 /*!
  * \var int printedLines
- * Lines printed since last user input (mod screen height)
+ * Lines printed since last user input (mod screen height).  This value is used
+ * to determine when a page of output is completed, and the user needs to be
+ * prompted for continuing output.  It counts the number of LF characters,
+ * which may differ from the lines actually used because some hard line breaks
+ * are enforced by overly long lines.
  */
-int printedLines = 0; /* Lines printed since last user input (mod screen height) */
+int printedLines = 0;
 flag g_scrollMode = 1; /* Flag for continuous (0) or prompted (1) scroll */
 flag g_quitPrint = 0; /* Flag that user quit the output */
 /*!
  * \var flag localScrollMode
  *
- * temporarily disables prompted scroll (see \ref g_scrollMode) until next user
- * prompt
+ * value 0: temporarily disables page-wise prompted scroll (see
+ * \ref g_scrollMode) until enabled (value 1) again.  Normal user input by
+ * \ref cmdInput1 allows scrolling during output (value 1).  A value of 0
+ * indicates the user has issued command _s_ or _S_ (see \ref pgBackBuffer)
+ * to print all pending output without interruption.  In this case the value 0
+ * is assigned temporarily to skip prompts.
+ */
+flag localScrollMode = 1;
+/*!
```

Comment:
```suggestion

/*!
```
Put a newline between definitions of more than one or two lines. The "block of declarations" style makes sense when the declaration and documentation are all in one line or maybe two, but if you add a big comment block then it just causes things to run togetner.